helmfile/charts/grafana-dashboards/files/welcome.md

```diff
@@ -4,11 +4,13 @@
 
 Here you can find the most relevant features and changes for the last couple of releases of Compliant Kubernetes
 
+- Harbor was upgraded to v2.9.1. As part of this upgrade, Notary V1 is removed. **[v0.35]**
+- Chroot for ingress-nginx is enabled by default. **[v0.35]**
```

Comment:
I wonder if the line about chroot should be reworded. I think it could be hard for app devs to understand what it means if nginx is running with chroot. Could you reword it to explain what effects it has for the app devs instead?
Same comment for the welcome dashboard in opensearch.

Reply:
pushed some changes